spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -69,6 +69,7 @@ content into your application. Rather, pick only the properties that you need.
 
 	# SPRING CORE
 	spring.beaninfo.ignore=true # Whether to skip search of BeanInfo classes.
+    spring.backgroundpreinitializer.ignore=true # Whether background preinitializer should be ignored.
```

Comment:
We shouldn't add this here as it's not managed the same way as a regular property. Let's document the property and its usage on the class Javadoc instead.